brax/envs/to_torch.py

```diff
@@ -0,0 +1,133 @@
+""" Wrapper around a Brax GymWrapper, that converts its outputs to PyTorch tensors.
+
+This conversion happens direcly on-device, witout the need to move values to the CPU.
+"""
+
+import datetime
+import time
+from collections import abc
+from functools import partial, singledispatch
+from typing import Any, Dict, Mapping, Optional, TypeVar, Union
+
+import gym
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from jax._src import dlpack as jax_dlpack
+from jaxlib.xla_extension import DeviceArray
+from torch import Tensor
+from torch.utils import dlpack as torch_dlpack
+
+from brax.envs.wrappers import GymWrapper, VectorGymWrapper
+
+
+class JaxToTorchWrapper(gym.Wrapper):
+    """Wrapper that converts Jax tensors to PyTorch tensors."""
+
+    def __init__(self, env: Union[GymWrapper, VectorGymWrapper], device: torch.device = None):
+        """Creates a Wrapper around a `GymWrapper` or `VectorGymWrapper` so it outputs
+        PyTorch tensors.
+
+        Parameters
+        ----------
+        env : Union[GymWrapper, VectorGymWrapper]
+            A `GymWrapper` or `VectorGymWrapper` to wrap.
+        device : torch.device, optional
+            device on which to move the Tensors. Defaults to `None`, in which case the
+            tensors will be on the same devices as their Jax equivalents.
+        """
+        if not hasattr(env, "reward_range"):
+            # NOTE: Fix a bug with the current GymWrapper, where the `Env` doesn't have
+            # the `reward_range` attribute which is needed by the gym.Wrapper
+            # constructor.
+            env.reward_range = (-np.inf, np.inf)
```

Comment:
If `GymWrapper` and `VectorGymWrapper` are missing an important attribute, by all means, add them there!  Thanks.

Reply:
I agree! I will set this as the default value in those classes for now. 
However, I wouldn't know how to determine that value from the wrapped Brax env, do you have any idea on how I could get the min reward and max rewards from it?

Reply:
Nevermind, it's actually not needed, since the `gym.Env` class has a default value for it. I'm not sure why I was getting an error before :monocle_face: